### ISSUE TITLE
Custom Fields

### ISSUE THREAD

**Issue by nickels** [trezor/trezor-password-manager#54]
It would be good to have the option to define custom fields and types (not only logins).

**Comment by chren-de-large** [comment]
Yes, we are planning a big refreshment of App itself but it will take time ...

**Comment by zsoltsandor** [comment]
I still second this, it would be nice to store some custom fields as well.
